src/cli/managers/config_manager.py

```diff
@@ -24,5 +24,40 @@ def get_applications(self) -> List[Dict[str, str]]:
     def application_exists(self, application: str) -> bool:
         return True
 
+    def remote_application_exists(self, application: str) -> Tuple[bool, int]:
+        """
+        Check if the application is already created on remote
+        by checking the remote_id field in the config file.
+        Returns True and application id if application exists on remote
+        Returns False and -1 if application does not exist on remote
+        """
+        application_info = self.get_application(application)
+        if 'remote_id' in application_info:
+            return True, int(application_info['remote_id'])
+
+        return False, -1
+
     def update_path(self, application: str, path: str) -> None:
         pass
+
+    def update_remote_id(self, application: str, application_id: int) -> None:
+        pass
+
+    def get_config_dir(self) -> Path:
+        return self.__config_dir
+
+    def remote_experiment_exists(self, path: Path) -> Tuple[bool, int]:
+        """
+        Check if the experiment is created on remote
+        by checking the experiment_id field in the meta attribute of experiment.json file.
+        Returns True and experiment id if experiment exists on remote
+        Returns False and -1 if experiment does not exist on remote
+        """
+        experiment_info = self.get_experiment(path)
```

Comment:
This will crash when you run it with the current implementation of get_experiment